log/log.go

```diff
@@ -143,13 +144,19 @@ func (m *defaultLogger) PrintBitriseStartedEvent(plan models.WorkflowRunPlan) {
 			EventType: "bitrise_started",
 		})
 	} else {
-		m.PrintBitriseASCIIArt(plan.Version)
-		m.Warnf("CI mode: %v", plan.CIMode)
-		m.Warnf("PR mode: %v", plan.PRMode)
-		m.Warnf("Debug mode: %v", plan.DebugMode)
-		m.Warnf("Secret filtering mode: %v", plan.SecretFilteringMode)
-		m.Warnf("Secret Envs filtering mode: %v", plan.SecretEnvsFilteringMode)
-		m.Warnf("No output timeout mode: %v", plan.NoOutputTimeoutMode)
+		m.Print("")
+		m.Printf("Invocation started at %s", colorstring.Cyan(m.opts.TimeProvider().Format(consoleTimeLayout)))
+		m.Printf("Bitrise CLI version: %s", colorstring.Cyan(plan.Version))
+		m.Print("")
+		m.Infof("Run modes:")
+		m.Printf("CI mode: %v", colorstring.Cyan("%v", plan.CIMode))
+		m.Printf("PR mode: %v", colorstring.Cyan("%v", plan.PRMode))
+		if plan.DebugMode {
+			m.Printf("Debug mode: %v", colorstring.Cyan("%v", plan.DebugMode))
+		}
```

Comment:
nit: I think it is perfectly fine to always print out if we are in debug mode or not.

Reply:
Yes, good point, if for nothing else, this introduces people to the debug flag 😆